javascript/javascript1/week1/1.js

```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
+
+const yearOfBirth = 1987;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
+);
+
+// Goodboy-oldboy excercise
+
+const dogYearOfBirth = 2017;
+
+let dogYearFuture = 2027;
+
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+let shouldShowResultInDogYears = dogYear;
```

Comment:
`shouldShowResultInDogYearsWe` should be boolean (either true or false), and here it is 10
The assignment says: _add an extra variable called shouldShowResultInDogYears. If it is true we should show the result in dog years, if it is false we should show it in human years._